processing/src/main/java/io/druid/query/aggregation/AggregatorUtil.java

```diff
@@ -84,4 +88,44 @@
     }
     return new Pair(condensedAggs, condensedPostAggs);
   }
+
+  public static FloatColumnSelector getFloatColumnSelector(
+      ColumnSelectorFactory metricFactory,
+      String fieldName,
+      String fieldExpression
+  )
+  {
+    if (fieldName != null) {
```

Comment:
Would be nice to verify here that exactly one of fieldName / fieldExpression is set, but not both.